### DISCUSSION TITLE
How to add support for new board (Cortex M7), similar like it is for STM43F4 Discovery?

### DISCUSSION THREAD

**Discussion by LAPSyLAB** [xpack-dev-tools/qemu-arm-xpack#23] [Q&A]
Are there any docs or info, how to add support for a new board (we have a large number of STM32H750DK boards (Cortex M7), that we would like emulate also visually as STM32F4 currently is already done ?

Thanks...

**Comment by ilg-ul** [comment]
Unfortunately qemu-system-arm does not implement graphical support for Cortex-M devices.

**Reply by LAPSyLAB** [reply]
No problem, graphical support is not necessary. Only plain common code execution.

**Reply by ilg-ul** [reply]
In this case I suggest you use the existing Cortex-M implementations available in QEMU.

For a functional project, please see <https://github.com/micro-os-plus/hello-world-qemu-template-xpack/> for details.
